test/pipeline_test.py

```diff
@@ -44,26 +44,74 @@ def test_drop():
 
     # Drop by columns
     df1 = DplyFrame(pandas_df)
-    output1 = df1 + drop(["col3", "col4"], axis=1)
-    expected1 = pandas_df.drop(["col3", "col4"], axis=1)
+    output1 = df1 + drop("col1", axis=1)
+    expected1 = pandas_df.drop("col1", axis=1)
```

Comment:
For this part, I was actually testing with different input parameters. First I was trying to use the positional arg (plus the axis), and then the columns keyword arg (without the axis arg), so I think it's better to keep it the way it was before.

Reply:
My output2 covers your previous output1